app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -587,6 +517,10 @@ class MainActivity : BottomSheetActivity(), ActionButtonActivity, MenuProvider {
         startActivity(composeIntent)
     }
 
+    override fun finish() {
+        super.finish()
+    }
```

Comment:
?

Reply:
Needed this to easily set a breakpoint for debugging, will remove, thx